extensions/lifecycle/objectProcessor/LifecycleObjectTransitionProcessor.js

```diff
@@ -102,8 +103,12 @@ class LifecycleObjectTransitionProcessor extends LifecycleObjectProcessor {
     getTask(actionEntry) {
         const actionType = actionEntry.getActionType();
 
-        if (actionType === 'requeueTransition') {
-            return new LifecycleResetTransitionInProgressTask(this);
+        switch (actionType) {
+            case 'requeueTransition':
+                return new LifecycleResetTransitionInProgressTask(this);
+            case 'requeueRestore':
+                return new LifecycleRetriggerRestoreTask(this);
+            default:
```

Comment:
what is the purpose of keeping the empty 'default' ?

Reply:
It is to be "exhaustive".. in fact it does nothing but I like it to be exhaustive in a switch case, anyway I don't mind removing it if you feel this is really useless tho

Reply:
Being "exhaustive" would be to cover each individual case explicitly ; and would make sense only with something done or at the very an explanation why we don't do anything.

Adding an empty `default:` is just telling the compiler/linter that you handle all cases, without actually doing anything : i.e. just blindsiding the tools (and reviewers)